front/src/modules/command-menu/components/CommandMenuItem.tsx

```diff
@@ -22,9 +30,10 @@ export const CommandMenuItem = ({
   onClick,
   Icon,
   shortcuts,
-}: CommandMenuItemProps) => {
```

Comment:
our new convention is to name the Props after the component name: so CommandMenuItemProps is correct